lua/ttt2/extensions/player.lua

```diff
@@ -0,0 +1,64 @@
+---
+-- player extension
+-- @author Mineotopia
+
+---
+-- Resets the spawn position of all players.
+-- @realm server
+function player.ResetSpawnPosition()
+	local plys = player.GetAll()
+
+	for i = 1, #plys do
+		plys[i]:SetSpawnPosition(Vector(0, 0, 0))
```

Comment:
Setting the position to 0,0,0 should not be considered a "reset" rather use nil, to differentiate between unset / set